target_snowflake/load_via_snowpipe.py

```diff
@@ -0,0 +1,107 @@
+import os
+import datetime
+import botocore
+from singer import get_logger
+from snowflake.connector.encryption_util import SnowflakeEncryptionUtil
+from snowflake.connector.remote_storage_util import SnowflakeFileEncryptionMaterial
+from snowflake.ingest import SimpleIngestManager, \
+    StagedFile
+from snowflake.ingest.utils.uris import DEFAULT_SCHEME
+from requests import HTTPError
+from cryptography.hazmat.primitives.serialization import load_pem_private_key, \
+    Encoding, \
+    PrivateFormat, \
+    NoEncryption
+from cryptography.hazmat.backends import default_backend
+from target_snowflake.s3_upload_client import S3UploadClient
+import target_snowflake.db_sync
+from snowflake.connector.errors import ProgrammingError
+# from snowflake.connector import DictCursor
+class LoadViaSnowpipe:
+
```

Comment:
if we can leave this better than we found it - some typing and a quick docstring as to what these args are would go a long way.
wtf is a dbLink?